lib/matplotlib/ticker.py

```diff
@@ -829,13 +829,13 @@ class LogFormatter(Formatter):
     """
     Format values for log axis.
     """
-    def __init__(self, base=10.0, labelOnlyBase=True):
+    def __init__(self, base=10.0, label_only_base=True):
```

Comment:
I think this fix has to hold off until we have python3 only and can do

```python
def __init__(self, base, label_only_base, * labelOnlyBase=none)
```

and then deprecate it